cmd/backend_kv_server.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright 2022 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/signal_set.hpp>
+#include <boost/process/environment.hpp>
+#include <CLI/CLI.hpp>
+#include <magic_enum.hpp>
+
+#include <silkworm/chain/config.hpp>
+#include <silkworm/common/log.hpp>
+#include <silkworm/rpc/backend_server.hpp>
+
+int main(int argc, char* argv[]) {
+    const auto pid = boost::this_process::get_id();
+    const auto tid = std::this_thread::get_id();
+
+    CLI::App app{"ETHBACKEND & KV servers"};
+
+    std::string chain_id{"goerli"};
+    std::string address_uri{"localhost:9090"};
+    uint32_t num_contexts{std::thread::hardware_concurrency() / 2};
+    std::string level_name;
+    app.add_option("--chain", chain_id, "The chain identifier as string", true);
+    app.add_option("--address", address_uri, "The address URI to bind the ETHBACKEND & KV services to", true);
+    app.add_option("--numContexts", num_contexts, "The number of running contexts", true);
+    app.add_option("--logLevel", level_name, "The log level identifier as string", true);
+
+    CLI11_PARSE(app, argc, argv);
+
+    const silkworm::ChainConfig* chain_config = silkworm::lookup_chain_config(chain_id);
+    if (chain_config == nullptr) {
+        SILK_CRIT << "Invalid chain identifier: " << chain_id;
+        return -1;
+    }
+
+    silkworm::log::Settings log_settings{};
+    log_settings.log_nocolor = true;
+    log_settings.log_threads = true;
+    const auto log_level = magic_enum::enum_cast<silkworm::log::Level>(level_name);
```

Comment:
The problem here is that the values in `log::Level` are prefixed with `k` (e.g. `kInfo`) according to [Google Style Guide](https://google.github.io/styleguide/cppguide.html#Enumerator_Names). I suggest to reuse the logic in `cmd::parse_silkworm_command_line` instead.